frontend/src/pages/devices/Detail.tsx

```diff
@@ -202,10 +202,10 @@ export const Detail = () => {
       }
 
       if (aValue === 4) {
-        return a.endAt.localeCompare(b.endAt)
```

Comment:
Just discovered that this should be flipped back the way it was... all sections have the "latest" item on top (continue to use `endAt` for completed items).

The comparison for the other sections should remain the way it is.